server/api/email.js

```diff
@@ -0,0 +1,185 @@
+'use strict';
+const Boom = require('boom');
+const Proposal = require('../models/proposal');
+const Feedback = require('../models/feedback');
+const User = require('../models/user');
+const Mailer = require('../mailer');
+const Config = require('../../config');
+var addDays = require('date-fns/addDays');
+var format = require('date-fns/format');
+
+const register = function (server, options) {  
+
+  server.route({
+    method: 'POST',
+    path: '/api/email/{proposalId}',
+    options: {
+      auth: {
+        strategies: ['simple', 'session'],
+        scope: ['reviewer', 'root', 'coordinator', 'chair']
+      }      
```

Comment:
Add validation for payload and check the allowed set of strings for template names.